packages/playwright-core/src/server/registry/nativeDeps.ts

```diff
@@ -463,18 +463,18 @@ export const deps: any = {
       'libnspr4',
       'libnss3',
       'libpango-1.0-0',
+      'libwayland-client0',
       'libx11-6',
       'libxcb1',
       'libxcomposite1',
       'libxdamage1',
       'libxext6',
       'libxfixes3',
       'libxkbcommon0',
-      'libxrandr2',
-      'libxshmfence1'
+      'libxrandr2'
     ],
     firefox: [
-      'ffmpeg',
```

Comment:
I have a vague memory that `ffmpeg` was required to run some videos, so can we keep it?